rpd_generator/bdl_structure/bdl_commands/circulation_loop.py

```diff
@@ -655,6 +656,9 @@ def determine_loop_flow_control(self):
             valve_type = circulation_loop.get_inp(
                 BDL_CirculationLoopKeywords.VALVE_TYPE_2ND
             )
+            """Jackson Q: When the current loop is a secondary loop with a two-way valve, shouldn't we
```

Comment:
went over this on the call together, but to put it in writing: When the current loop is a secondary loop with a two-way valve it only means that the Primary loop serving the current loop is variable flow